src/lch.rs

```diff
@@ -1,6 +1,9 @@
+use num::traits::Float;
+
 use std::ops::{Add, Sub};
 
-use {Color, ColorSpace, Mix, Shade, GetHue, Hue, Rgb, Luma, Xyz, Lab, Hsv, Hsl, Saturate, LabHue, clamp};
+use {Color, ColorSpace, Mix, Shade, GetHue, Hue, Rgb, Luma, Xyz, Lab, Hsv, Hsl, Saturate, LabHue,
+     clamp};
```

Comment:
Hah! I would rather have it completely vertical or grouped. No big deal, though :smile: